### ISSUE TITLE
Incorrect interpreter path in ./node_modules/.bin/gdaxConsole

### ISSUE THREAD

**Issue by ty2u** [coinbase/gdax-tt#11]
Hi there,

This looks great!

I was going to test and see how it works but the first step failed:

```
$ ./node_modules/.bin/gdaxConsole --product BTC-USD -t
bash: ./node_modules/.bin/gdaxConsole: /bin/env: bad interpreter: No such file or directory
```

Changing the file to use /usr/bin/env instead of /bin/env fixed it. I noticed all the other files in that same folder are already using /usr/bin/env

**Comment by CjS77** [comment]
Thanks for pointing this out. The demos changed just before launch and clerly forgot to updte the docs.

**Comment by CjS77** [comment]
Fixed in v0.1.14
